### PR TITLE
[ci-skip] Add README.md files

### LINKED ISSUE #2284: feat(corda-ent): update platform helm charts with README.md
### Description
As a developer I want to add proper documentation for the helm charts, before they are packaged and published on bevel helm repo

### Acceptance Criteria
1. README.md file added.
2. vaules.yaml file filled with default values.
3. chart.yaml versions updated.

### PR DESCRIPTION
**Pull request by sailajakommineni**
Primary Changes
--------------
Add readme files for all charts
Update the values.yaml files
Update the version in chart.yaml file

Modifications in charts
---------------------
platforms/r3-corda-ent/charts/auth
platforms/r3-corda-ent/charts/bridge
platforms/r3-corda-ent/charts/float
platforms/r3-corda-ent/charts/gateway
platforms/r3-corda-ent/charts/generate-pki-node
platforms/r3-corda-ent/charts/generate-pki
platforms/r3-corda-ent/charts/h2
platforms/r3-corda-ent/charts/idman
platforms/r3-corda-ent/charts/nmap
platforms/r3-corda-ent/charts/node-initial-registration 
platforms/r3-corda-ent/charts/node
platforms/r3-corda-ent/charts/notary-initial-registration 
platforms/r3-corda-ent/charts/notary
platforms/r3-corda-ent/charts/signer
platforms/r3-corda-ent/charts/zone

Fixes #2284

### CODE REVIEW

File: platforms/r3-corda-ent/charts/auth/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+### Config
+
+| Name                     | Description                                                                          | Default Value   |
+| ------------------------ | -------------------------------------------------------------------------------------| --------------- |
+| baseDir                  | Provide volume related specifications                                                | "/opt/corda"    |
+| jarPath                  | Provide the path where the CENM Idman .jar-file is stored                            | "bin"           |
+| configPath               | Provide the path where the CENM Service configuration files are stored               | "etc"           |
+| pvc                      | Provide any extra annotations for the PVCs                                           | ""              |
+| deployment               | Provide any extra annotations for the deployment                                     | "vaule"         |
```
Comment (jagpreetsinghsasan) [inline platforms/r3-corda-ent/charts/auth/README.md:128]: `vaule` typo

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]
There are lots of empty fields, please fix them all

File: platforms/r3-corda-ent/charts/generate-pki-node/values.yaml [hunk 2]
```diff
@@ -28,13 +28,13 @@ image:
   # Provide the alpine utils image, which is used for all init-containers of deployments/jobs.
   # NOTE: The alpine image used is the base alpine image with CURL installed.
   # Eg. initContainerName: hyperledgerlabs/alpine-utils:1.0
-  initContainerName:
+  initContainerName:
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/generate-pki-node/values.yaml:31]: have default values, check the same for all charts

File: platforms/r3-corda-ent/charts/generate-pki-node/README.md [hunk 3]
```diff
@@ -3,4 +3,170 @@
...
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "Generate-pki-node-deployment-helm-chart"></a>
+## Generate-pki-node Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/main/platforms/r3-corda-ent/charts/generate-pki-node) generates the certificates and configure a PKI for node.
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/generate-pki-node/README.md:20]: generates the corda complaint certificate hierarchy for the node

File: platforms/r3-corda-ent/charts/generate-pki/Chart.yaml [hunk 4]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for generating the certificates
+description: "R3-corda-ent: Generates the pki-certificates."
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/generate-pki/Chart.yaml:9]: deploys the pki for cenm services

File: platforms/r3-corda-ent/charts/generate-pki/README.md [hunk 5]
```diff
@@ -3,4 +3,188 @@
...
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "Generate-pki-deployment-helm-chart"></a>
+## Generate-pki Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/main/platforms/r3-corda-ent/charts/generate-pki) generates the certificates and configure a PKI for a CENM application.
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/generate-pki/README.md:20]: deploys the pki to generate the corda complaint certificate hierarchy for the CENM services

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]

File: platforms/r3-corda-ent/charts/generate-pki-node/Chart.yaml [hunk 6]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for generating the certificates
+description: "R3-corda-ent: deploys the pki."
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/generate-pki-node/Chart.yaml:9]: capital 'D' for deploys

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]
Rest is fine

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**